up.go

```diff
@@ -794,8 +843,23 @@ var (
 	whiteOnDBlue = tcell.StyleDefault.Foreground(tcell.ColorWhite).Background(tcell.ColorNavy)
 )
 
-func drawText(region Region, style tcell.Style, text string) {
-	for x, ch := range text {
-		region.SetCell(x, 0, style, ch)
+func drawText(region Region, x int, style tcell.Style, text string) {
```

Comment:
Please try changing the last parameter from `string` type to `[]rune`. I think this may simplify most of the calls to drawText in the new code; if I see correctly, there might be only one place where an explicit `[]rune(...)` conversion will be needed when calling drawText.

Reply:
Could you review 243eb17 ?

I modified drawText function as you pointed out. 
* type of the last parameter
* return value as a lengh of text 
* use a seq variable to hold a sequence of characters